pilot/pkg/security/authz/builder/testdata/all-fields-in.yaml

```diff
@@ -15,10 +15,12 @@ spec:
             requestPrincipals: ["requestPrincipals", "requestPrincipals-prefix-*", "*-suffix-requestPrincipals", "*"]
             namespaces: ["ns", "ns-prefix-*", "*-ns-suffix", "*"]
             ipBlocks: ["1.2.3.4", "5.6.0.0/16"]
+            remoteIpBlocks: ["1.2.3.4", "5.6.0.0/16"]
```

Comment:
Can you also add the new fields (remote_ip, not_remote_ip and "remote.ip" attribute) in the following test files:
action-audit-HTTP-for-TCP-filter-in.yaml 
action-deny-HTTP-for-TCP-filter-in.yaml
single-policy-in.yaml